tests/test_samples.py

```diff
@@ -0,0 +1,154 @@
+from tests.test_base import *
+from tests.test_value_constants import *
+from tests.mock_vtservice import MockVtService
+
+class TestSamples(BaseClientTest):
+
+    def test_basicdomainreport_example(self):
+        # Modify sample file to include necessary sample data
+        sample_filename = self.BASIC_FOLDER + "/basic_domain_report_example.py"
+
+        with self.create_client(max_retries=0) as dxl_client:
+            # Set up client, and register mock service
+            dxl_client.connect()
+
+            with MockVtService(dxl_client):
+                mock_print = BaseClientTest.run_sample(sample_filename)
+
+                mock_print.assert_any_call(
+                    StringDoesNotContain("Error")
+                )
+
+                # Validate whois_timestamp from report
+                mock_print.assert_any_call(
+                    StringContains(
+                        str(SAMPLE_DOMAIN_REPORT["whois_timestamp"])
+                    )
+                )
+
+            dxl_client.disconnect()
+
+
+    def test_basicfilereport_example(self):
+        # Modify sample file to include necessary sample data
+        sample_filename = self.BASIC_FOLDER + "/basic_file_report_example.py"
+
+        with self.create_client(max_retries=0) as dxl_client:
+            # Set up client, and register mock service
+            dxl_client.connect()
+
+            with MockVtService(dxl_client):
+                mock_print = BaseClientTest.run_sample(sample_filename)
+
+                mock_print.assert_any_call(
+                    StringDoesNotContain("Error")
+                )
+
+                # Validate whois_timestamp from report
```

Comment:
I think the comment above should say something about the `md5` instead of the `whois_timestamp`. Looks like this should be updated in all of the related tests below this one as well.

Reply:
Comments updated.